database/server.go

```diff
@@ -504,8 +504,8 @@ func GetServiceName() (string, error) {
 }
 
 // Retrieve all service names from most recent to oldest
-func GetServiceNameHistory() ([]server_structs.ServiceName, error) {
-	var entries []server_structs.ServiceName
+func GetServiceNameHistory() ([]server_structs.ServerName, error) {
```

Comment:
Should the name of this function and its docstring change to match the fact that it takes as input a server name, not a service name?